app/views/pictures/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<% if @pictures.any? %>
+  <% @pictures.each do |picture| %>
+    <div><%= image_tag picture.image.url(:thumb) %></div>
+    <div><h2><%= link_to picture.name, picture_path(picture) %></h2></div>
+    <div><%= link_to "Like Picture", picture_likes_path(picture), class: 'likes-link' %>
+     <span class="likes_count"> <%= picture.likes.count %> </span> <%= 'likes'.pluralize(picture.likes.count) %></div>
+    <div><%= link_to "Add A Comment #{picture.name}", new_picture_comment_path(picture) %> </div>
+    <div> <%= link_to "Edit #{picture.name}", edit_picture_path(picture) %>
+      <%= link_to "Delete #{picture.name}", picture_path(picture), method: :delete %></div>
+    <h3>Comments for <%= picture.name %></h3>
+    <% if picture.comments.any? %>
+      <ul>
+        <% picture.comments.each do |comment| %>
+          <li>
+            <%= comment.comment %>
+          </li>
+        <% end %>
+      </ul>
+    <% else %>
+      <p>No comments.</p>
+    <% end %>
+  <% end %>
+<% else %>
+  No pictures yet
+<% end %>
+<div><h2><a href='pictures/new'>Add A Picture</a></h2></div>
```

Comment:
Again I feel like some of this could be made more comprehensible with the use of partials:

```erb
<% if @pictures.any? %>
  <% @pictures.each do | picture | %>
     <%= render partial: 'picture', locals: { picture: picture } %>
  <% else %>
  <p>No pictures.</p>
<% end %>
```

Then inside the `_picture.erb` partial, you could have a `_comment.erb` partial too.